UnicornRender/video/source/vulkan/Renderer.cpp

```diff
@@ -1375,26 +1387,26 @@ bool Renderer::CreatePipelineCache()
     vk::PipelineCacheCreateInfo pipelineCacheCreateInfo;
     if(m_vkLogicalDevice.createPipelineCache(&pipelineCacheCreateInfo, nullptr, &pipelineCache) != vk::Result::eSuccess)
     {
-        LOG_ERROR( "Can't create pipeline cache!" );
+        spdlog::get(utility::Logger::LoggerName())->error( "Can't create pipeline cache!" );
         return false;
     }
     return true;
 }
 
 bool Renderer::LoadEngineHelpData()
 {
-    unicorn::video::Texture texture;
+    auto texture = std::make_shared<Texture>();
```

Comment:
Why make it `shared_ptr` when it's only used in this scope?

Reply:
VkTexture::Create have std::shared_ptr as input parameter which is calls below

Reply:
Why does `VkTexture::Create` take a `shared_ptr` as parameter? Does it store it inside? Does it share the ownership because it may be deleted while it's working with it?